path_config.go

```diff
@@ -145,14 +146,16 @@ func (b *azureSecretBackend) pathConfigWrite(ctx context.Context, req *logical.R
 
 	if useMsGraphApi, ok := data.GetOk("use_microsoft_graph_api"); ok {
```

Comment:
I thought about adding logic to prevent users from going back to using AAD graph once this has been switched to true. That might be a little too forceful though. Happy to hear thoughts around this.

Reply:
It does sound a bit forceful. My opinion would be to let the operator decide.